### PR TITLE
fixes #209

### LINKED ISSUE #209: Add search cache back in
Once we can reduce the payload size of the page (less than 2MB), then we can revert #206

### PR DESCRIPTION
**Pull request by lzrscg**

### CODE REVIEW

File: apps/web/lib/headless-utils.ts [hunk 1]
```diff
@@ -93,17 +115,17 @@ export async function loadIntegration(networkSlug: string) {
...
   context?: PaginationContext;
-  skipCache?: boolean;
 }): Promise<Page> {
   const integration = await loadIntegration(route.network);
 
   const fixedPath = parseHeadlessRouteVercelFix(route).path;
 
-  const resolution = await integration.resolveRoute(fixedPath, context, true);
+  const resolution = await integration.resolveRoute(fixedPath, context, {
+    skipCache: true,
```
Comment (lzrscg) [inline apps/web/lib/headless-utils.ts:127]: @fredk3 I actually think we shouldn't skip cache here. if we do that, then I think the page load (after the search redirect) will not pull from cache. thoughts?

Comment (Fredkiss3) [reply apps/web/lib/headless-utils.ts:127]: yeah, we shouldn't need to skip the cache. i just skipped it here out of convenience.

Comment (fredk3) [reply apps/web/lib/headless-utils.ts:127]: yeah, we shouldn't need to skip the cache. i just skipped it here out of convenience.

Comment (lzrscg) [reply apps/web/lib/headless-utils.ts:127]: ok perfect! I will remove this

**Pull request merged**